io/http/html post request dispatcher/88ec6117-e970-47ca-81f6-70c633880e85.json

```diff
@@ -0,0 +1,172 @@
+{
+  "name": "io/http/HTML post request dispatcher",
```

Comment:
I'm a bit hesitant regarding the name, especially the term "HTML post" as technically HTML does not define anything like "post".
Should it rather be "HTML form post" as in the description?

Reply:
I'd also put HTTP methods in all-caps as a convention (POST, GET, etc.)